WasatchNET/Spectrometer.cs

```diff
@@ -1462,7 +1513,34 @@ public bool laserRampingEnabled
         }
         // bool laserRampingEnabled_;
 
-        public virtual bool areaScanEnabled
+        public virtual UInt16 laserWatchdogSec
+        {
+
+            get
+            {
+                const Opcodes op = Opcodes.GET_LASER_WATCHDOG_SEC;
+                if (haveCache(op))
+                    return laserWatchdogSec_;
+                readOnce.Add(op);
+                return laserWatchdogSec_ = Unpack.toUshort(getCmd2(op, 2));
+            }
+            set
+            {
+                const Opcodes op = Opcodes.GET_DETECTOR_START_LINE;

```

Comment:
This doesn't look right **(change requested)**

Reply:
This probably needs resolved before merging

Reply:
good catch